### DISCUSSION TITLE
2022/09/25/2022-sept-cf-tutorials-1/

### DISCUSSION THREAD

**Discussion by giscus** [DeCalvin2006/blog#6] [General]
# 2022/09/25/2022-sept-cf-tutorials-1/

最近做了不少 CF 上的 DP 题（PS：标签事 dp，但是实际上却是各种各样的东西）。 可能要写很多篇分着写QwQ，争取在9月结束之前写完吧。  1706 B. Making Towers 题面链接 (Rating:1100)  比较有技术含量的一道题。  给定 n 个方块，最多有nnn种不同的颜色。每次可以选一个方块将其放在上、左、右三个方向。定义塔为yyy轴方向上连续的一段同色小方块。对于1

https://decalvin.netlify.app/2022/09/25/2022-sept-cf-tutorials-1/

**Comment by DeCalvin2006** [comment]
先暂时写两篇~
欢迎催更QwQ
